static/js/ajax-notifications.js

```diff
@@ -15,10 +15,13 @@ PNotify.prototype.options.styling = 'jqueryui';
 PNotify.prototype.options.width = '340px';
 PNotify.desktop.permission();
 
-function displayPNotify(type, title, message) {
+function displayPNotify(type, title, message, id) {
```

Comment:
In python its preferred to not use buildins like `id`. So you can easily solve this, by rename it to notification_id or something like that.